moxy/org.eclipse.persistence.moxy/pom.xml

```diff
@@ -87,12 +77,10 @@
         <dependency>
             <groupId>com.sun.activation</groupId>
             <artifactId>jakarta.activation</artifactId>
-            <scope>runtime</scope>
         </dependency>
         <dependency>
             <groupId>com.sun.mail</groupId>
             <artifactId>jakarta.mail</artifactId>
-            <scope>runtime</scope>
         </dependency>
```

Comment:
On JDK 11, there it does not matter. Only API classes are exported from these implementations, so usage of implementation class would lead to compile time error (not a case when on classpath or JDK 8 an older). Other than that keeping the clear separation here would bring additional changes on few other places (mainly surefire would need to exclude `-api` artifacts from paths to avoid collisions). This may be revisited once there is new version of activation (and mail) available at least for testing